ecl/hql/hqlexpr.ipp

```diff
@@ -39,6 +39,7 @@
 #include "javahash.hpp"
 #include "defvalue.hpp"
 #include "hqlexpr.hpp"
+#include "hqlgram.hpp"
```

Comment:
Why is this needed?  If it is the definition of the function I would prefer the function to be defined in hqlexpr.cpp.  The visibility/dependence on the parser should be as small as possible.